bindata/assets/alerts/client-go-requests.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: client-go
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+    - name: control-plane-client-go
+      rules:
+        - record: client_go_request_duration:method_get:histogram_quantile99
```

Comment:
we could make the recording rule generic here by adding a quantile label as in: https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/cluster-monitoring-operator/prometheus-rule.yaml#L420-L423. This would allow having multiple `cluster:rest_client_request_duration_seconds:histogram_quantile` and select which one we want to use by specifying the quantile label. In our case
```
cluster:rest_client_request_duration_seconds:histogram_quantile{quantile="0.99"}
```

Reply:
added